cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -675,7 +675,7 @@ SwiftGenerator::typeToString(const TypePtr& type, const ContainedPtr& usedBy, bo
         "Swift.Float",
         "Swift.Double",
         "Swift.String",
-        "Ice.Disp",      // Object
+        "Ice.Value",     // Object
```

Comment:
```suggestion
        "Ice.Value",     // Object not used anymore
```

Reply:
`Object` actually remains a valid syntax in Slice (it's an alias for `Value`).

But we apparently don't test it in Swift.

Reply:
In slice2cs it has this "not used anymore" comment.

https://github.com/zeroc-ice/ice/blob/14159b77b284a769bc513ef792dbfd9798f3d410/cpp/src/slice2cs/CsUtil.cpp#L127

Reply:
You're right, it's no longer used because of the logic in this function. Fixed comment.
```cpp
    if (builtin)
    {
        if (builtin->kind() == Builtin::KindObject)
        {
            t = getUnqualified(builtinTable[Builtin::KindValue], currentModule);
        }
        else
        {
            t = getUnqualified(builtinTable[builtin->kind()], currentModule);
        }
    }
```